packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -91,6 +96,20 @@ class AnalyzeContinuously extends AnalyzeBase {
         }
       }
 
+      // Summarize dartdoc issues rather than displaying each individually
+      int membersMissingDocumentation = 0;
+      if (flutterRepo && !showDartDocIssuesIndividually) {
+        errors.removeWhere((AnalysisError error) {
```

Comment:
This is a personal preference, but I think it's easier to reason about code when there are fewer mutations. Here instead of mutating the `errors` list, and therefore making the logic sensitive to the order in which operations happen, we could instead create new lists of errors whose names describe what's in the list, e.g.:

```dart
bool isMissingDocError(AnalysisError error) => error.code == 'public_member_api_docs';

final List<AnalysisError> allErrors = ...;
final int membersMissingDocumentation = allErrors.where(isMissingDocError).length;
final int detailedErrors = allErrors.where((e) => !isMissingDocError(e)).toList()..sort();
```